dev-embedded/arduino-builder/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>junk4me46806@yahoo.com</email>
+	</maintainer>
+</pkgmetadata>
```

Comment:
use ```metagen``` and fill out more fields https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/